scripts/requirements.txt

```diff
@@ -1,3 +1,12 @@
 GitPython==3.1.27
 pyyaml==5.4
-requests==2.27.1
\ No newline at end of file
+requests==2.27.1
+tqdm==4.63.0
+azure-ai-ml
+azure-identity
+transformers[torch]
+wget
+pandas
+mlflow
+https://scorestorageforgeneric.blob.core.windows.net/libs/azureml_metrics-0.1.0.72521629-py3-none-any.whl
+https://scorestorageforgeneric.blob.core.windows.net/libs/azureml_evaluate_mlflow-0.1.0.74451692-py3-none-any.whl
```

Comment:
Could we please not use static package urls here? We could expose a feed or wait till this gets pushed to PyPi? 

@ShipraJain01 What do you think?

Reply:
Same as above.